copy_to_g.sh

```diff
@@ -0,0 +1,4 @@
+cp dist/arethusa.min.js ~/workspace/Gardener/_assets/arethusa
```

Comment:
I really like these scripts. They make it easier to test changes in the applications that use the Arethusa Widget. The one thing I'm not sure about is hardcoding `~/workspace`.

What do you think about either having them copy to some directory in this repository (e.g. `./dist_widget`), taking a command line argument for the location, or using some other kind of configuration?

Reply:
oops didn't mean to commit these scripts but yes, they do make it easier to test. Your suggestion is a good one. I can make that change.

Reply:
Well I'm glad you did commit them. It would have taken me longer to get it working without them.

Reply:
ok, there is now a deploy_widget.sh script which will copy the files to a directly passed to the script, or else to dist_widget directory